src/explore-education-statistics-admin/src/validation/validation.ts

```diff
@@ -3,12 +3,23 @@ import {
   dayMonthYearIsComplete,
   dayMonthYearIsEmpty,
 } from '@common/services/publicationService';
-import { isValid } from 'date-fns';
+import { isValid, parse } from 'date-fns';
 
 const yupNumberOrUndefinedIfBlank = Yup.mixed().transform((val: string) =>
   val === '' ? undefined : parseInt(val, 10),
 );
 
+export const parseDate = (
+  value: { year: number; month: number; day: number },
+  backupDate?: Date,
```

Comment:
Think you could make this a default of `new Date(value.year, value.month, value.day)`